src/current/_includes/v24.1/sidebar-data/self-hosted-deployments.json

```diff
@@ -366,6 +366,12 @@
                     "urls": [
                       "/${VERSION}/essential-metrics-self-hosted.html"
                     ]
```

Comment:
Not for this PR — as you add more pages to the Monitoring and Alerting nav in the future, I wonder if it would help to add sub dropdowns to start grouping the pages.